src/features/deliveryStatus/fetchDeliveryStatus.ts

```diff
@@ -24,12 +24,13 @@ import {
 const PROCESS_TOPIC_0 = '0x1cae38cdd3d3919489272725a5ae62a4f48b2989b0dae843d3c279fee18073a9';
 
 export async function fetchDeliveryStatus(
+  multiProvider: MultiProvider,
   message: Message,
-  multiProvider = getMultiProvider(),
 ): Promise<MessageDeliveryStatusResponse> {
   const destName = multiProvider.getChainName(message.destinationChainId);
+  const destEnv = getChainEnvironment(multiProvider, destName);
   // TODO PI support here
-  const destMailboxAddr = hyperlaneCoreAddresses[destName]?.mailbox;
+  const destMailboxAddr = hyperlaneEnvironments[destEnv][destName]?.mailbox;
```

Comment:
Looking at this code, should we even separate by `environments`? It seems like more and more like a relict of v1?

Reply:
We shouldn't, this is because asa removed the hyperlaneCoreAddresses object in the SDK. I plan to add it back soon